protocols/v2/subprotocols/mining/src/close_channel.rs

```diff
@@ -6,20 +6,15 @@ use binary_sv2::{Deserialize, Serialize, Str0255};
 #[cfg(not(feature = "with_serde"))]
 use core::convert::TryInto;
 
-/// # CloseChannel (Client -> Server, Server -> Client)
+/// Message used by a downstream to close a mining channel.
 ///
-/// Client sends this message when it ends its operation. The server MUST stop sending messages
-/// for the channel. A proxy MUST send this message on behalf of all opened channels from a
-/// downstream connection in case of downstream connection closure.
+/// If you are sending this message through a proxy on behalf of multiple downstreams, you must send
+/// it for each open channel separately.
 ///
-/// If a proxy is operating in channel aggregating mode (translating downstream channels into
-/// aggregated extended upstream channels), it MUST send an UpdateChannel message when it
-/// receives CloseChannel or connection closure from a downstream connection. In general, proxy
-/// servers MUST keep the upstream node notified about the real state of the downstream
-/// channels.
+/// Upon receiving this message, upstream **must** stop sending messages for the channel.
```

Comment:
by looking at the rest of the PR, it seems that the overall pattern is to:
- clean up and summarize spec language
- make docs as simple as possible

with that in mind, I feel that these comments above `CloseChannel` are still a bit too verbose

> /// Message used by a downstream to close a mining channel.

this single line seems enough for Rust Docs

maybe we should consider chopping off these extra comment lines with guidelines of "must" do things?

the reader should find that kind of info when they're reading the specs

Reply:
anyways not strongly opinionated on this... fine the way it is now as well